pynest/examples/CampbellSiegert.py

```diff
@@ -204,11 +202,11 @@ def psp(x):
 # omitted so initial transients do not perturb our results. We then print the
```

Comment:
Above, I'd suggest remove the `float()` in
`syn_spec={'weight': float(J[indx]), 'delay': 0.1}
`

Reply:
done